etl/android-clients.kp/knowledge.md

```diff
@@ -161,11 +161,11 @@ while day <= end:
             StructField("defaultsearch", StringType(), True),
             StructField("device", StringType(), True),
             StructField("arch", StringType(), True),
-            StructField("fennecActivityStreamTopsitesLoaderTimeMs", 
+            StructField("fennec_activity_stream_topsites_loader_time_ms", 
```

Comment:
no, only snake_casing camelCased ones. This should keep most queries/notebooks relevant with minimal changes.